test/NuGet.Core.Tests/NuGet.Frameworks.Test/FrameworkNameProviderTests.cs

```diff
@@ -121,7 +121,7 @@ public void FrameworkNameProvider_EquivalentFrameworkPrecedence()
             // Arrange
             var mappingsA = new Mock<IFrameworkMappings>();
             var mappingsB = new Mock<IFrameworkMappings>();
-            mappingsA.Setup(x => x.EquivalentFrameworkPrecedence).Returns(new[] { Windows });
+            mappingsA.Setup(x => x.EquivalentFrameworkPrecedence).Returns(new[] { FrameworkConstants.FrameworkIdentifiers.Windows });
```

Comment:
Why do you need namespace here?

Reply:
`Windows` was ambiguous after upgrading the VS Package version (assuming there's a new type with the same name).